### DISCUSSION TITLE
window 训练特别慢

### DISCUSSION THREAD

**Discussion by xbkaishui** [PaddlePaddle/PaddleOCR#12970] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win10 22H2
- 版本号/Version：Paddle 2.6：  PaddleOCR： 2.6.1.0 问题相关组件/Related components：train 
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

在window 运行paddle 训练， 200张图， 一个epoch 需要100s ， batchsize 是 16， window上 gpu利用率也比较低 
同样的数据在 linux 只需要 几秒

请教下问题出在哪里？

**Comment by xbkaishui** [comment]
@tink2123  any update ?

**Comment by TingquanGao** [comment]
两台机器CPU、内存规格一致吗？

**Comment by xbkaishui** [comment]
linux的 cpu好些， gpu都是3090，我理解这个主要是耗时在gpu上吧

**Comment by xbkaishui** [comment]
@TingquanGao  @tink2123  可否加个微信 18267139686 , 愿意付费请教下
